WordPress/src/main/res/drawable/ic_login_magic_link_sent.xml

```diff
@@ -0,0 +1,55 @@
+<vector android:height="120dp" android:viewportHeight="160.0"
```

Comment:
The naming convention used for vector resources is `ic_` for icons/small/simple graphics and `img_` for images/large/complex graphics.  Also, resources are named for what they are rather than how they're used (e.g. `ic_plus_white_24dp` rather than `ic_editor_fab`) that way they can be reused intuitively.  What do you think about renaming this `img_email_alert_120dp` or something like that?